doc/releases/release-notes-3.7.rst

```diff
@@ -407,6 +412,10 @@ Libraries / Subsystems
 
   * Mbed TLS was updated to 3.6.0. Release notes can be found at:
     https://github.com/Mbed-TLS/mbedtls/releases/tag/v3.6.0
+  * When any PSA crypto provider is available in the system, desired PSA features
+    must be explicitly selected through ``CONFIG_PSA_WANT_xxx`` symbols. Moreover
+    TLS/X509/PK/MD modules will make use of PSA APIs instead of legacy Mbed TLS
+    ones.
```

Comment:
> Moreover TLS/X509/PK/MD modules will make use of PSA APIs instead of legacy Mbed TLS ones.

This was already the case at least in the `CONFIG_MBEDTLS_PSA_CRYPTO_C` case: https://github.com/zephyrproject-rtos/zephyr/pull/71827/files#diff-35af03c2e234822e54b1a5ef8918c465ca47f246d1dc732740e125e6a2935732R475-R479